addons/point_of_sale/models/pos_order.py

```diff
@@ -66,10 +66,18 @@ def _get_valid_session(self, order):
                         closed_session.id,
                         order['name'],
                         order['amount_total'])
+        rescue_session = PosSession.search([
+            ('name', '=like', '%(RESCUE FOR %(session)s)' % {'session': closed_session.name}),
+            ('state', 'not in', ('closed', 'closing_control')),
+        ], limit=1)
+        if rescue_session:
+            _logger.warning('reusing recovery session %s for saving order %s', (rescue_session.name, order['name']))
```

Comment:
Good point, I missed that. But please do not interpolate yourself, let logging do it for you, as you intended.

In fact this should have made the tests fail, perhaps we should extend the coverage to the rescue flow...